ext/mbstring/libmbfl/mbfl/mbfl_consts.h

```diff
@@ -31,18 +31,14 @@
 #ifndef MBFL_CONSTS_H
 #define MBFL_CONSTS_H
 
-#define MBFL_ENCTYPE_SBCS		0x00000001
-#define MBFL_ENCTYPE_MBCS		0x00000002
-#define MBFL_ENCTYPE_WCS2BE		0x00000010
-#define MBFL_ENCTYPE_WCS2LE		0x00000020
-#define MBFL_ENCTYPE_MWC2BE		0x00000040
-#define MBFL_ENCTYPE_MWC2LE		0x00000080
-#define MBFL_ENCTYPE_WCS4BE		0x00000100
-#define MBFL_ENCTYPE_WCS4LE		0x00000200
-#define MBFL_ENCTYPE_MWC4BE		0x00000400
-#define MBFL_ENCTYPE_MWC4LE		0x00000800
-#define MBFL_ENCTYPE_SHFTCODE	0x00001000
-#define MBFL_ENCTYPE_ENC_STRM	0x00002000
+#define MBFL_ENCTYPE_SBCS		0x00000001 /* single-byte encoding */
+#define MBFL_ENCTYPE_MBCS		0x00000002 /* multi-byte encoding */
+#define MBFL_ENCTYPE_WCS2BE		0x00000010 /* 2 bytes/char, high byte first */
```

Comment:
I believe just using the established technical terms here is clearer:
```suggestion
#define MBFL_ENCTYPE_WCS2BE		0x00000010 /* 2 bytes/char, big endian */
```
I don't insist though.

Reply:
I like it.